templates/index.html

```diff
@@ -34,6 +35,7 @@
             <td>{{ resource.allocated }}</td>
             <td>{{ resource.allocated_to_id }}</td>
             <td>{{ resource.allocated_to_address }}</td>
+            <td>{{ resource.allocated_at }}</td>
```

Comment:
OK, result now displays as an int? I think python probably has a fucntion to datify if so.